specs/SUMMARY.md

```diff
@@ -53,6 +54,7 @@
   - [Fungible Token](Standards/FungibleToken/README.md)
       - [Core Standard](Standards/FungibleToken/Core.md)
       - [Metadata](Standards/FungibleToken/Metadata.md)
+      - [Events](Standards/FungibleToken/Event.md)
```

Comment:
Same here. Oh, BTW, please add Multi-Token links to this file, it's required for Nomicon (links will not work in other cases, I struggle with it each time #307 #309)